src/modules/dart.rs

```diff
@@ -139,4 +140,25 @@ mod tests {
         assert_eq!(expected, actual);
         dir.close()
     }
+
+    #[test]
+    fn detect_version_output_in_stdout() -> io::Result<()> {
+        // after dart 2.15.0, version info output in stdout.
+        let dir = tempfile::tempdir()?;
+        File::create(dir.path().join("any.dart"))?.sync_all()?;
+
+        let actual = ModuleRenderer::new("dart")
+            .cmd(
+                "dart --version",
+                Some(CommandOutput {
+                    stdout: String::from("Dart SDK version: 2.15.1 (stable) (Tue Dec 14 13:32:21 2021 +0100) on \"linux_x64\""),
```

Comment:
Thanks! But I believe this should have a trailing newline.
```suggestion
                    stdout: String::from("Dart SDK version: 2.15.1 (stable) (Tue Dec 14 13:32:21 2021 +0100) on \"linux_x64\"\n"),
```

Reply:
But there is no new line in utils.rs
https://github.com/starship/starship/blob/10e89121595827ce99042d036b1ee55f8a3a6a08/src/utils.rs#L128-L133

Reply:
You're right, I must have been mistaken.